src/lib/app-helper.js

```diff
@@ -81,6 +82,23 @@ async function runPackageScript (scriptName, dir, cmdArgs = []) {
   }
 }
 
+/** @private */
+async function runPackageScriptDetached (scriptName, dir, cmdArgs = []) {
+  if (!dir) {
+    dir = process.cwd()
+  }
+  aioLogger.debug(`running npm run-script ${scriptName} in dir: ${dir}`)
+  const pkg = await fs.readJSON(path.join(dir, 'package.json'))
+  if (pkg && pkg.scripts && pkg.scripts[scriptName]) {
+    const command = pkg.scripts[scriptName].split(' ')
+
+    const child = aioRunDetached.run(command, dir)
```

Comment:
This is my attempt at running a detached process from aio app run
1. aioRunDetached needs to take a dir to cwd into
2. aioRunDetached.run should return the process so that it can be killed on exit

Reply:
This has since been changed and I think this pr needs to be updated or closed.

Reply:
Ping!! @rajarju ^^